release-content/0.14/release-notes/11426_Computed_State__Sub_States.md

```diff
@@ -1 +1,107 @@
-TODO
+Bevy's `States` are a simple but powerful abstraction for managing the control flow of your app.
+
+```rust
+use bevy_state::prelude::*;
+use bevy_ecs::prelude::IntoSystemConfigs;
+use bevy_ecs::system::ResMut;
+
+
+#[derive(Clone, Copy, PartialEq, Eq, Hash, Debug, Default, States)]
+enum GameState {
+ #[default]
+  Menu,
+  InGame,
+}
+
+fn handle_escape_pressed(mut next_state: ResMut<NextState<GameState>>) {
+    if escape_pressed {
+        next_state.set(GameState::Menu);
+    }
+}
+
+fn open_menu() {
+    // Show the settings menu...
+}
+
+App::new()
+    .init_state::<GameState>()
+    .add_systems(Update, handle_escape_pressed.run_if(in_state(GameState::Menu)));
```

Comment:
Should this say `in_state(GameState::InGame)` or do I just not understand Bevy states?

Reply:
Yes, it should.